packages/components/tests/dummy/app/templates/components/form/base-elements.hbs

```diff
@@ -141,6 +141,174 @@
 
   <Shw::Divider />
 
+  <Shw::Text::H2>Character count</Shw::Text::H2>
```

Comment:
I had difficulties following the examples in this part of the showcase. If you don't mind, I am going to send you a patch (impossible to do a suggestion here) with how I would reorganize (and simplify) the code examples here, to see if works better for you too.

Reply:
Applied the patch in ef3dcef